### PR TITLE
Add support for old multi-interface syntax.

### PR DESCRIPTION
**Pull request by ruiaraujo**
There is not obvious disadvantage to support so I don't think it requires an option.

### CODE REVIEW

File: test/runner.js [hunk 1]
```diff
@@ -278,48 +278,92 @@ describe('Runner', () => {
...
+      assert.equal(
+        'The field `Comment.author` is missing a description.',
+        errors[9].message
+      );
+      assert.equal(3, errors[9].location.line);
+      assert.equal(
+        `${__dirname}/fixtures/schema/comment.graphql`,
+        errors[9].location.file
+      );
+      assert.equal(errors[9].rule, 'fields-have-descriptions');
```
Comment (ruiaraujo) [inline test/runner.js]: I think a migration towards Jest and snapshots would save a lot of maintenance time. 😄

**Comment by cjoudrey** [comment]
Thanks again for the contribution @ruiaraujo! 😄Good catch, I completely overlooked this behavior when bumping `graphql`.

> There is not obvious disadvantage to support so I don't think it requires an option.

While there is nothing wrong with the way you implemented it, I'd prefer having an option for this because the _right_ way of doing this is using `&` and [this backwards compatibility option will eventually disappear](https://github.com/graphql/graphql-js/blob/5fe39262a308df944a87cc85b225228e7556aaa4/src/language/parser.js#L89-L97).

By forcing users to opt-in to this behavior using an option, they know that they need to fix this. By silently accepting it, we are not encouraging users to move to the `&` syntax.

This reminds me a bit of PR #95 which you can use as a reference to add a new option.

File: test/runner.js (hunk 1 above)
Comment (cjoudrey) [reply test/runner.js]: Can you elaborate on this? 😄 I'm not super familiar with Jest and snapshots.

**Comment by ruiaraujo** [comment]
I disagree with the need for a new option when we can transparently support both use cases. When upgrading the dependency to a version that does not support the old version, it would be trivial to remove this.

Another option I would suggest, is a warning in case the user is using the old syntax. Also if people use prettier they get upgraded to the new syntax once [this](https://github.com/prettier/prettier/pull/4012) is merged.

What do you think of my suggestion?

**Comment by cjoudrey** [comment]
> I disagree with the need for a new option when we can transparently support both use cases.

The problem I have with supporting both syntax transparently is that users have no way of enforcing one way or the other. If I take the time to migrate an entire schema to use the new `&` syntax and someone accidentally uses `,` somewhere, I want the linter to complain and not silently accept it.

This is why I'm suggesting the default behavior should always be the latest spec, but add a way for people to gracefully upgrade to this. 

An opposing view to this (which I _think_ you are advocating for) is whether the `graphql-schema-linter` should care about this syntax in the first place.

> Also if people use prettier they get upgraded to the new syntax once [this](https://github.com/prettier/prettier/pull/4012) is merged.

That's excellent! 😀

**Comment by barry-dutton-ck** [comment]
Entering this convo a little late but... I encountered an issue of support with this in my project, but couldn't work around it (as we are using `graphql@0.10.0`). If I changed it to the proper value in my schema for linting, it would not compile when I started the app. It would be great if this was supported until fully deprecated for _comilation_ purposes, but then fired off a level of linting error which could be specified (`warn` or `error`) to encourage people to upgrade but not prevent them from linting their schemas at all.

**Comment by cjoudrey** [comment]
Just circling back to this PR, I've decided to add `--old-implements-syntax` in #124 so I will be closing this PR.

@ruiaraujo I appreciate the initial stab you took at this problem. I'd also still love to see your take on the tests using Jest and snapshots, but I think this can be done in another PR.

**Comment by ruiaraujo** [comment]
Thanks for finishing this. It has been on my TODO list for a while. 😄 

When I have time I will do a Jest migration.

**Pull request closed without merging**